flake.nix

```diff
@@ -162,6 +162,15 @@
                 alejandra -c .
               '';
             };
+
+            shfmt = simpleCheck rec {
+              name = "shell-checks";
+              src = pkgs.lib.sources.sourceFilesBySuffices ./. [".sh"];
+              nativeBuildInputs = [pkgs.shfmt];
+              checkPhase = ''
+                shfmt -d -s -i 4 -ci ${src}
+              '';
+            };
```

Comment:
If this is an option it needs to be a CI check (part of ./contrib/lint.sh?) otherwise we're gonna get fat diffs with lots of formatting changes between diffs that don't run shfmt

Reply:
just track it